### PR TITLE
update: fetch list desa digital

### PR DESCRIPTION
**Pull request by yoslie**
## Changes
- update fetch list data desa digital level one, level two, level three, and level four
- minor wording table head

## Evidence
title: Update fetch list data desa digital level one, level two, level three, and level four
project: Desa Digital
participants: @yoslie @doohanas @maruf12 @agunghide

### CODE REVIEW

File: components/DesaDigital/LevelOne/index.vue [hunk 1]
```diff
@@ -42,7 +48,46 @@ export default {
...
+    try {
+      const response = await this.$axios.get('/villages/list-with-location', { params: this.query })
+      const { data, meta } = response.data
+      this.data = data || []
+      this.pagination.currentPage = meta?.current_page || 1
+      this.pagination.totalRows = meta?.total || 0
+      this.pagination.itemsPerPage = meta?.per_page || this.query.per_page
+    } catch (error) {
+      this.pagination.disabled = true
+    }
```
Comment (doohanas) [inline components/DesaDigital/LevelOne/index.vue]: Ask:

Hello @yoslie, I wondering why to put the `error` as a parameter in the catch but It's never used? 

#cmiiw

**Review by doohanas — APPROVED** [review]

**Review by maruf12 — APPROVED** [review]

File: components/DesaDigital/LevelOne/index.vue (hunk 1 above)
Comment (yoslie) [reply components/DesaDigital/LevelOne/index.vue]: open discuss:
Just to make sure it's catch an error lol,
actually I don't know is if it's must be used or not. But different from the declaring variable inside the function, `eslint` didn't see it as unused variable so I don't remove it just as a sign that it catch an error. #cmiiw 
But I can remove it if u want 😄

Comment (doohanas) [reply components/DesaDigital/LevelOne/index.vue]: During the param in catch block is never used and we have statements to handle any exceptions. So, we can remove that. wdyt? 😃

Comment (yoslie) [reply components/DesaDigital/LevelOne/index.vue]: sure changes [here](https://github.com/jabardigitalservice/dedi-cms/pull/88/commits/e317d7ef6f2b7f86dba669ce42680f368e9af732) thankyou 🙏

**Review by agunghide — APPROVED** [review]

**Pull request merged**